.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,4 +4,8 @@
   You will need to follow these steps:
   1. Edit sigs.yaml with your change 
   2. Generate docs with `make generate`. To build docs for one sig, run `make WHAT=sig-apps generate`
--->
\ No newline at end of file
+-->
+
+<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->
+Fixes/Refers #
```

Comment:
If we are going to put this in, please use the same language from the kubernetes/kubernetes repo:
https://github.com/kubernetes/kubernetes/blame/master/.github/PULL_REQUEST_TEMPLATE.md#L26-L32

Reply:
Makes sense.